src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -205,21 +188,6 @@ public override void Initialize (AnalysisContext context)
 				foreach (var extraSymbolAction in ExtraSymbolActions)
 					context.RegisterSymbolAction (extraSymbolAction.Action, extraSymbolAction.SymbolKind);
 
-				void CheckAttributeInstantiation (
```

Comment:
Why is it OK to remove this?
Where is the place we perform this check now? (and why did it change with a new analyzer package version)

Reply:
There is new callback calls in attributes, for at least one scenario (I didn't test all scenarios) instead of going through a type argument it went through an object creation, basically when it sees the use of the attribute it created a callback for object creation and then we have a CheckMemberCall that verifies that it has requires producing the warning. My guess is that it was a limitation of the analyzer that we had to work around, just like the global attribute and at somepoint it got fixed

Reply:
OK - I guess if all the tests are passing, it's at least not completely broken